src/FrontDoor/FrontDoor/Cmdlets/NewFrontDoorWafPolicy.cs

```diff
@@ -90,7 +90,13 @@ public class NewFrontDoorWafPolicy : AzureFrontDoorCmdletBase
         /// </summary>
         [Parameter(Mandatory = false, HelpMessage = "Custom Response Body")]
         public string CustomBlockResponseBody { get; set; }
-
+        
+        /// <summary>
+        /// The tags to associate with the Front Door Waf Policy.
+        /// </summary>
+        [Parameter(Mandatory = false, HelpMessage = "The tags associate with the Front Door Waf Policy.")]
+        public Hashtable Tag { get; set; }
```

Comment:
Please update the help docs for New-AzFrontDoorWafPolicy

Reply:
done